File: benchmarks/classes/Bench.java
public class Bench {
    static class Person {
        int age;
        int weight;
        Person(int age, int weight) {
            this.age = age;
            this.weight = weight;
        }
        int getValue() {
            return age + weight;
        }
    }

    public static void main(String[] args) {
        long sum = 0;
        for (int i = 0; i < 1000000; i++) {
            Person p = new Person(i, i + 1);
            sum += p.getValue();
        }
        System.out.println(sum);
    }
}

File: benchmarks/classes/bench.dart
class Person {
  int age;
  int weight;
  Person(this.age, this.weight);
  int getValue() => age + weight;
}

void main() {
  int sum = 0;
  for (int i = 0; i < 1000000; i++) {
    var p = Person(i, i + 1);
    sum += p.getValue();
  }
  print(sum);
}
